gammapy/modeling/models/tests/test_io.py

```diff
@@ -335,9 +335,27 @@ def make_all_models():
         norms=[3, 4] * u.cm,
     )
     yield Model.create("GaussianSpectralModel", "spectral")
-    # TODO: yield Model.create("EBLAbsorptionNormSpectralModel")
-    # TODO: yield Model.create("NaimaSpectralModel")
-    # TODO: yield Model.create("ScaleSpectralModel")
+    yield Model.create(
+        "EBLAbsorptionNormSpectralModel",
+        "spectral",
+        energy=[0, 1, 2] * u.keV,
+        param=[0, 1],
+        data=np.ones((2, 3)),
+        redshift=0.1,
+        alpha_norm=1.0,
+    )
+    import naima
```

Comment:
Here it would be better to have a separate test for the Naima moldel creation with the decorator @requires_dependency("naima")